components/about_build_modal/__snapshots__/about_build_modal.test.jsx.snap

```diff
@@ -340,17 +373,50 @@ exports[`components/AboutBuildModal should match snapshot for team edition 1`] =
         </a>
       </div>
       <div
-        className="form-group about-modal__copyright"
+        className="form-group"
       >
-        <FormattedMessage
-          defaultMessage="Copyright 2015 - {currentYear} Mattermost, Inc. All rights reserved"
-          id="about.copyright"
-          values={
-            Object {
-              "currentYear": 2017,
+        <div
+          className="about-modal__copyright"
+        >
+          <FormattedMessage
+            defaultMessage="Copyright 2015 - {currentYear} Mattermost, Inc. All rights reserved"
+            id="about.copyright"
+            values={
+              Object {
+                "currentYear": 2017,
+              }
             }
-          }
-        />
+          />
+        </div>
+        <div
+          className="about-modal__links"
+        >
+          <a
+            href="http://about.mattermost.com/default-terms/"
+            rel="noopener noreferrer"
+            target="_blank"
+          >
+            <FormattedMessage
+              defaultMessage="Terms of Service"
+              id="about.tos"
+              values={Object {}}
+            />
+          </a>
+          <span>
+             - 
+          </span>
+          <a
+            href="http://mattermost.com/privacy-policy/"
```

Comment:
And similarly here, SupportSettings. PrivacyPolicyLink in config.json

Reply:
Same reason as previous comment.